__tests__/articles.test.js

```diff
@@ -45,4 +45,81 @@ describe("GET/api/articles/:article_id", () => {
 
         expect(returnedArticle).toEqual(expectedArticle)
     })
+})
+
+
+describe("GET/api/articles", () => {
+    test("200 - responds with a 200 status and an array of all 13 topics", async () => {
+        const result = await request(app).get("/api/articles").expect(200)
+        const articlesArray = result.body.articles
+
+        expect(Array.isArray(articlesArray)).toBe(true)
+        expect (articlesArray.length).toBe(13)
+    })
+
+    test("200 - all articles have the appropriate keys / values", async () => {
+        const result = await request(app).get("/api/articles").expect(200)
+        const articlesArray = result.body.articles
```

Comment:
Really thorough testing but maybe split out into too many tests. You could see about just making the one request and then testing everything within one forEach loop. This would reduce this down to 2 tests, one for this and another for testing the order?